lib/android/src/main/java/com/airbnb/android/react/maps/AirMapPolylineManager.java

```diff
@@ -68,6 +72,26 @@ public void setZIndex(AirMapPolyline view, float zIndex) {
     view.setZIndex(zIndex);
   }
 
+  @ReactProp(name = "lineCap")
+  public void setlineCap(AirMapPolyline view, String lineCap) {
+    Cap cap = null;
+    switch (lineCap) {
+      case "butt":
+        cap = new ButtCap();
+        break;
+      case "round":
+        cap = new RoundCap();
+        break;
+      case "square":
+        cap = new SquareCap();
+        break;
+      default:
+        cap = new RoundCap();
```

Comment:
Dow e need a default cap ? I mean maybe people don't want one.

Reply:
the docs say round is the default one

Reply:
ok great, I'm blind 😹

Reply:
:D